README.md

```diff
@@ -3,32 +3,32 @@ vscode extension for container related secure supply chain tools
 
 ## Features
 
-Describe specific features of your extension including screenshots of your extension in action. Image paths are relative to this README file.
+The Show Referrers feature makes it easy to access and view the referrers and artifacts of a image. Although the referrers will appear in the same structure as the output of the command `oras discover OCI_ARTIFACT`.
 
-For example if there is an image subfolder under your extension project workspace:
+![Docker extension overview](resources/readme/ezgif.com-optimize.gif)
 
-\!\[feature X\]\(images/feature-x.png\)
+## Requirements
+First run 'npm install' to get necessary libraries.
 
-> Tip: Many popular extensions utilize animations. This is an excellent way to show off your extension! We recommend short, focused animations that are easy to follow.
+Be sure to be logged into your Azure container registries and Docker client through terminal.
```

Comment:
Can we be explicit about what this means in terms of steps? Does the user need to run `az acr login` on the cli. Do they need to do `docker login`? Or do they need to press some other button in the extension?